dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -1982,8 +1982,8 @@ CREATE TABLE `t_ds_cluster`(
 DROP TABLE IF EXISTS `t_ds_fav_task`;
 CREATE TABLE `t_ds_fav_task`
 (
-    `id`        bigint      NOT NULL AUTO_INCREMENT COMMENT 'favorite task id',
-    `task_name` varchar(64) NOT NULL COMMENT 'favorite task name',
+    `id`        bigint      NOT NULL AUTO_INCREMENT COMMENT 'id',
+    `task_type` varchar(64) NOT NULL COMMENT 'favorite task type name',
```

Comment:
does this feature already release? If released we should also add this incompatible change into https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/upgrade/incompatible.html

Reply:
This feature has released in 3.1.0, we need to add DDL

Reply:
So it is mean this featrue should work in 3.1.0 but currently not work, am I right?

Reply:
> So it is mean this featrue should work in 3.1.0 but currently not work, am I right?

Dear Jiajie, this is an upgrade to make the column name more appropriate. It works without this pr changes.

Reply:
The users upgrade by upgrade tools, and it will error if we don't provide this DDL sql.

Reply:
> does this feature already release? If released we should also add this incompatible change into https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/upgrade/incompatible.html

have submitted dml changes to mysql and postgre